oppia/oppia-android
# Fix part of #1460: Json file updates for questions and skills

## Explanation


This PR fixes part of #1460 

## Checklist
- [x] All questions are shifted to a single `questions.json` file.
- [x] All questions are shifted to a single `skills.json` file.

## Pending
- [ ] Structure of question json files needs to be updated. (Will be done in future PR)
- [ ] Topic Size Calculation.

## How to test:
1. All tests are passing without major change makes sure that the content is parsing correctly.
2. Run the app, go to `Fractions`-> `Practice`, start questions.
3. Make `ConceptCardFragmentTestActivity` as launcher activity and check that it works.

## Checklist
